### PR TITLE
SAK-48650 Trinity: Display problem for site descriptions with links

### PR DESCRIPTION
**Pull request by hornersa**

### CODE REVIEW

File: portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeSiteNav.vm [hunk 1]
```diff
@@ -169,6 +171,18 @@
 
             document.querySelector('.Mrphs-portalWrapper').focus();
         }, 50);
+
+        const siteDescPopoverList = Array.prototype.slice.call(document.querySelectorAll('[data-bs-toggle="popover"]'));
+        siteDescPopoverList.map((el) => {
```
Comment (adrianfish) [inline portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeSiteNav.vm:176]: map isn't the right thing to use here, imo. Map returns a new array with the mapped elements in it, which you don't need here. I'd just use forEach.

File: portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeSiteNav.vm [hunk 2]
```diff
@@ -169,6 +171,18 @@
 
             document.querySelector('.Mrphs-portalWrapper').focus();
         }, 50);
+
+        const siteDescPopoverList = Array.prototype.slice.call(document.querySelectorAll('[data-bs-toggle="popover"]'));
```
Comment (adrianfish) [inline portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeSiteNav.vm:175]: You can just use spread here. It does the same thing and is more concise. In fact, if you use spread, you can probably get rid of the variable and have the whole thing on one line.
[...document.querySelectorAll('[data-bs-toggle="popover"]')].forEach ...

File: portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeSiteNav.vm [hunk 3]
```diff
@@ -169,6 +171,18 @@
...
+        siteDescPopoverList.map((el) => {
+            let opts = {
+                animation: false,
+            }
```
Comment (adrianfish) [inline portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeSiteNav.vm:179]:
```suggestion
            };
```

**Review by adrianfish — CHANGES_REQUESTED** [review]

**Comment by hornersa** [comment]
@adrianfish - Thanks for the pointers. I'll force push a revision.

**Comment by hornersa** [comment]
I inadvertently messed up this branch, so I'll attempt a different PR.

**Pull request closed without merging**